mixer/deploy/kube/conf/mixer-dashboard.json

```diff
@@ -0,0 +1,1857 @@
+              "show": true
+            }
+          ]
+        },
+        {
+          "aliasColors": {},
+          "bars": false,
+          "dashLength": 10,
+          "dashes": false,
+          "datasource": "${DS_PROMETHEUS}",
+          "fill": 1,
+          "id": 36,
+          "legend": {
+            "avg": false,
+            "current": false,
+            "max": false,
+            "min": false,
+            "show": true,
+            "total": false,
+            "values": false
+          },
+          "lines": true,
+          "linewidth": 1,
+          "links": [],
+          "nullPointMode": "null",
+          "percentage": false,
+          "pointradius": 5,
+          "points": false,
+          "renderer": "flot",
+          "seriesOverrides": [],
+          "spaceLength": 10,
+          "span": 3,
+          "stack": false,
+          "steppedLine": false,
+          "targets": [
+            {
+              "expr": "irate(envoy_cluster_mixer_server_upstream_rq_retry[1m])",
+              "format": "time_series",
+              "intervalFactor": 2,
+              "legendFormat": "Total",
+              "refId": "A"
+            },
+            {
+              "expr": "irate(envoy_cluster_mixer_server_upstream_rq_retry_success[1m])",
+              "format": "time_series",
+              "intervalFactor": 2,
+              "legendFormat": "Success",
+              "refId": "B"
+            },
+            {
+              "expr": "irate(envoy_cluster_mixer_server_upstream_rq_retry_overflow[1m])",
+              "format": "time_series",
+              "intervalFactor": 2,
+              "legendFormat": "Overflow",
+              "refId": "C"
+            }
+          ],
+          "thresholds": [],
+          "timeFrom": null,
+          "timeShift": null,
+          "title": "Client Retries",
+          "tooltip": {
+            "shared": true,
+            "sort": 0,
+            "value_type": "individual"
+          },
+          "type": "graph",
+          "xaxis": {
+            "buckets": null,
+            "mode": "time",
+            "name": null,
+            "show": true,
+            "values": []
+          },
+          "yaxes": [
+            {
+              "format": "short",
+              "label": null,
+              "logBase": 1,
+              "max": null,
+              "min": null,
+              "show": true
+            },
+            {
+              "format": "short",
+              "label": null,
+              "logBase": 1,
+              "max": null,
+              "min": null,
+              "show": false
+            }
+          ]
+        }
+      ],
+      "repeat": null,
+      "repeatIteration": null,
+      "repeatRowId": null,
+      "showTitle": false,
+      "title": "Dashboard Row",
+      "titleSize": "h6"
+    },
+    {
+      "collapse": false,
+      "height": "40",
+      "panels": [
+        {
+          "content": "<center><h2>Adapters and Config</h2></center>",
```

Comment:
curious: should we split config and adapter section ? Is there specific reason you think grouping these is useful ?

Reply:
This was just a collection point for high-level, Mixer-specific metrics. When the rest of the runtime2 metrics come online, I think we should split these out into separate sections.